school/src/main/java/com/hogwarts/school/controller/FacultyController.java

```diff
@@ -18,16 +19,15 @@ public FacultyController(FacultyService facultyService) {
         this.facultyService = facultyService;
     }
 
-    @PostMapping("/addFaculty")
+    @PostMapping("addFaculty")
```

Comment:
```suggestion
    @PostMapping
```
не нужно ничего писать в пути - это ошибка REST